src/pymovements/gaze/screen.py

```diff
@@ -47,15 +47,22 @@ class Screen:
         Eye-to-screen distance in centimeters
     origin : str
         Specifies the screen location of the origin of the pixel coordinate system.
-    x_max_dva : float
```

Comment:
Removing this is not an option.

Reply:
#583 refactored `x_max_dva` etc. into a property method. The docstring here is then probably not needed anymore.